gridcapa-task-manager-api/src/test/java/com/farao_community/farao/gridcapa/task_manager/api/ProcessFileDtoTest.java

```diff
@@ -25,4 +30,32 @@ void testConstructorEmptyProcessFile() {
         assertNull(processFileDto.getLastModificationDate());
         assertEquals(ProcessFileStatus.NOT_PRESENT, processFileDto.getProcessFileStatus());
     }
+
+    @Test
+    void testEqualsAndHashCode() {
+        OffsetDateTime now = OffsetDateTime.now();
+        ProcessFileDto referenceDto = new ProcessFileDto("path/to/filename", "FILETYPE", ProcessFileStatus.VALIDATED, "filename", now);
+        ProcessFileDto equalDto = new ProcessFileDto("path/to/filename", "FILETYPE", ProcessFileStatus.VALIDATED, "filename", now);
+        ProcessFileDto diffFilePathDto = new ProcessFileDto("other/path/to/filename", "FILETYPE", ProcessFileStatus.VALIDATED, "filename", now);
+        ProcessFileDto diffFileTypeDto = new ProcessFileDto("path/to/filename", "DIFFERENT", ProcessFileStatus.VALIDATED, "filename", now);
+        ProcessFileDto diffStatusDto = new ProcessFileDto("path/to/filename", "FILETYPE", ProcessFileStatus.NOT_PRESENT, "filename", now);
+        ProcessFileDto diffFilenameDto = new ProcessFileDto("path/to/filename", "FILETYPE", ProcessFileStatus.VALIDATED, "other", now);
+        ProcessFileDto diffDateDto = new ProcessFileDto("path/to/filename", "FILETYPE", ProcessFileStatus.VALIDATED, "filename", now.minusMinutes(1));
+
+        assertFalse(referenceDto.equals(null)); // do not use assertFalse, which does not call equals method in this case
```

Comment:
in the comment you put do not use "assertFalse...."   and you actually used it !